src/MercadoPago/MercadoPagoExtendSocialite.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace SocialiteProviders\MercadoPago;
+
+use SocialiteProviders\Manager\SocialiteWasCalled;
+
+class MercadoPagoExtendSocialite
+{
+    public function handle(SocialiteWasCalled $socialiteWasCalled): void
+    {
+        $socialiteWasCalled->extendSocialite('mercadolibre', Provider::class);
```

Comment:
Wrong?

Reply:
yes, it should be 'mercadopago' instead, the other comments about scope and identifier i just used in the same way MercadoLibre is using, maybe could be edited for both